src/lib/UtapiClient.js

```diff
@@ -169,33 +167,26 @@ export default class UtapiClient {
      */
     _getParamsArr(params) {
         this._checkMetricTypes(params);
+        const props = [];
+        const { byteLength, newByteLength, oldByteLength, numberOfObjects } =
+            params;
+        const metricData = {
```

Comment:
We cannot pass `params` here because this function creates isolated objects for each metric level ('buckets', 'accounts', etc.). The `params` object can contain both levels (for example, how S3 passes an object with 'bucket' and 'accountId' as properties). I've now added a more explicit comment in the JSDoc here.